resources/views/components/game/leaderboards-listing/leaderboards-list-item.blade.php

```diff
@@ -19,6 +19,10 @@ class="leading-3"
 
         @php
             $bestScore = $leaderboard->entries()
+                ->whereHas('user', function ($query) {
+                    $query->where('Untracked', '!=', 1)
+                        ->whereNull('unranked_at');
+                })
```

Comment:
Shouldn't this just be part of `entries()`? Would we ever want to return untracked users?

It looks like `ActsAsPlayer::scopeTracked()` and the `GetLeaderboardData` query filter out untracked users unless they're the logged in user. I think we should revisit that logic too. If a user is untracked, we could potentially show where they would be in the leaderboard, but the leaderboard rankings for tracked users shouldn't be shifted when an untracked user views the list.

Reply:
Agreed, doing it on `entries()` is probably the right call. I've updated `entries()` in latest to account for this. It also accepts an optional boolean arg in the event we do want those untracked records.

For showing where a user would theoretically be placed on the leaderboard, I don't have a terribly strong opinion. It might be easier to implement after we migrate leaderboardinfo.php wholesale from the legacy pages.